compiler/rustc_passes/src/check_attr.rs

```diff
@@ -2141,6 +2153,7 @@ fn check_invalid_crate_level_attr(tcx: TyCtxt<'_>, attrs: &[Attribute]) {
         sym::rustc_main,
         sym::unix_sigpipe,
         sym::derive,
+        sym::manually_drop,
```

Comment:
i don't think this should be necessary :thinking: 

what happens if you don't add it here and use `manually_drop` as a crate attr?

Reply:
You're right! That's simple enough. It's already checked by the other changes to this file, so removing it from this list makes the error messages less redundant.